www/js/album.js

```diff
@@ -1,14 +1,51 @@
+let albumDialogOpen;
+
 $$(document).on('page:init', '.page[data-name="album"]', function (e) {
   var albumId = e.detail.route.params.albumId;
+  albumDialogOpen = false;
   $.getJSON(API + '/albums/' + albumId)
     .done(function(resp) {
       initAlbums(resp.album);
     })
     .fail(function(resp) {
-      console.log(resp.statusText);
+      $$('.album-content .preloader').remove();
+      if (resp.statusText === 'Unauthorized') {
+        albumDialog.open();
+      } else {
+        $$('.album-content').append('<div class="block">Kunde inte läsa in album</div>');
+      }
     });
 });
 
+const albumDialog = app.dialog.create({
```

Comment:
Rename to fit the behavior better and don't block a "normal" dialog and its name for the future.
```suggestion
const albumAuthDialog = app.dialog.create({
```